backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/ProprietaryController.java

```diff
@@ -4,12 +4,12 @@
 
 import de.bund.digitalservice.ris.norms.adapter.input.restapi.mapper.ProprietaryResponseMapper;
 import de.bund.digitalservice.ris.norms.adapter.input.restapi.schema.ProprietarySchema;
+import de.bund.digitalservice.ris.norms.application.exception.NormNotFoundException;
```

Comment:
Did you remove the try/catch in the endpoints of this controller (and the other controllers, if present) for the `NormNotFoundException`? We don't need them anymore because we are handling now this exception also in the controller advice.